Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -6617,6 +6617,28 @@ These ItemGrids are available in the API and can be manipulated by the plugins,
 					},
 					Notes = "Removes one item from the hotbar's currently selected slot. Returns true on success.",
 				},
+				ReplaceOneEquippedItem =
+				{
+					Params =
+					{
+						{
+							Name = "Item",
+							Type = "cItem",
+						},
+						{
+							Name = "TryOtherSlots",
```

Comment:
(minor) Perhaps it would be worth holding off exporting this for now and waiting to see if anyone requests it? Only that the code doesn't set the parameter right now.

Reply:
I believe this function is such a basic functionality, that it warrants an immediate export.

Reply:
oh I mean the "TryOtherSlots" parameter

Reply:
Nah, keep it.